lib/travis/api/v3/queries/log.rb

```diff
@@ -8,7 +8,7 @@ def find(job)
       raise EntityMissing, 'log not found'.freeze if log.nil?
       #if the log has been archived, go to s3
       if log.archived_at
-        content = fetch.get(prefix).try(:body)
+        content = fetch.get(prefix).try(:body).force_encoding('UTF-8')
```

Comment:
Can you explain what garbage bytes are and how to feed them to a log.

The coding was enforced due to random logs having  'xE2/' in the string which was throwing errors. See: https://sentry.io/travis-ci/org-api-staging/issues/216159260/